config/auth_actions.php

```diff
@@ -59,13 +59,15 @@
             'check_username',
             'check_email',
             'for_language',
+            'login_dialog_template',
         ],
         'vocabulary' => [ 'of' ],
         'wall' => [
             'index',
             'show_message',
             'messages_of_user',
         ],
+        'angular_templates' => '*',
```

Comment:
Could you write this as `'angular_templates' => [ '*' ]` instead? This is because newer versions of the cake-auth-actions plugin will require this syntax. See the yet-to-be-merged commit 266c5162f8e402e906736d7da4a5149ea1ea9f6f.

Reply:
I'm getting a `403 Forbidden` if I put the wildcard in an array. I guess that change will have to be done at the same time as upgrading to the newer version?

Reply:
I see. My bad :pray: